test/functional/page_objects/visual_builder_page.ts

```diff
@@ -226,10 +239,40 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
     public async changeDataFormatter(
       formatter: 'Bytes' | 'Number' | 'Percent' | 'Duration' | 'Custom'
     ) {
-      const [formatterEl] = await find.allByCssSelector('.euiComboBox');
+      const formatterEl = await find.byCssSelector('[id$="row"] .euiComboBox');
```

Comment:
I believe it's worth to specify the selector:
```suggestion
      const formatterEl = await find.byCssSelector('[id$="date-row"] .euiComboBox');
```

Reply:
it failed on ci
![image](https://user-images.githubusercontent.com/8816260/59383944-b6d4e980-8d69-11e9-8d00-eccb2735a10b.png)